indico/web/templates/forms/email_recipients_widget.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'forms/base_widget.html' %}
+
+{% block html %}
+    <div class="recipients-box left" {% if field.errors %} data-error="{{ _render_errors(field.errors) }}"{% endif %}>
+        <div class="text">
+            {{- field._value()|markdown -}}
```

Comment:
Because in that field it makes some sense (so the text can be easily formatted) - for a list of emails? Not so much.